index.html

```diff
@@ -0,0 +1,278 @@
+<html>
+  <head>
+    <title>
+      Manubot - Publishing, reinvented.
+    </title>
+    <meta
+      name="description"
+      content="The next generation of scholarly publishing: open, collaborative, reproducible, free."
+    />
+    <meta
+      name="keywords"
+      content="scholarly, publishing, publisher, manuscript, document, academic, education, open, collaborative, reproducible, free, edit, collaborate, ideas, version control, code, writing, automatic, digital, interactive, print, fair"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Quicksand"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <style></style>
+  </head>
+  <body>
+    <header>
+      <div class="logo_container">
+        <img class="logo logo_shadow" src="logo-animated.svg" />
+        <img class="logo" src="logo-animated.svg" />
+      </div>
+      <div class="logo_text">
+        Manubot
+      </div>
+      <div class="logo_subtext">
+        Publishing, reinvented.
+        <span class="cursor_blink">|</span>
+      </div>
+    </header>
+    <section class="special_section">
+      <div class="row flex_wrap">
+        <a href="" class="button_link">
+          <i class="fas fa-graduation-cap"></i>
+          Get Started
+        </a>
+        <a href="" class="button_link">
+          <i class="fas fa-lg fa-book"></i>
+          Docs
+        </a>
+        <a href="" class="button_link">
+          <i class="fab fa-lg fa-github"></i>
+          On GitHub
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-asterisk fa-lg"></i> At a glance
+          </div>
+          <div>
+            Manubot is a workflow and set of tools for the next generation of
+            scholarly publishing. Write your manuscript in
```

Comment:
Can we do one sentence per line, since we're writing and reviewing this HTML file directly?